src/components/Badge/styles.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import { theme } from '../../theme';
+
+export const RoundedBadge = styled.div`
+  display: inline-block;
+  border-radius: ${theme.borderRadius.xl};
```

Comment:
Do you plan to make the border radius a prop on the component?

Reply:
No.. but I can ! Indeed we talked about that :load: